awx/ui/client/src/inventories-hosts/inventories/related/groups/list/groups-list.controller.js

```diff
@@ -102,35 +102,51 @@
             $state.go('inventories.edit.groups.edit.nested_groups', {group_id: id});
         };
         $scope.deleteGroup = function(group){
-            $scope.toDelete = {};
-            $scope.strings.deleteModal = {};
-            angular.extend($scope.toDelete, group);
-            if($scope.toDelete.total_groups === 0 && $scope.toDelete.total_hosts === 0) {
-                // This group doesn't have any child groups or hosts - the user is just trying to delete
-                // the group
-                $scope.deleteOption = "delete";
-            }
-            else {
-                $scope.strings.deleteModal.group = InventoryHostsStrings.get('deletegroup.GROUP', $scope.toDelete.total_groups);
-                $scope.strings.deleteModal.host = InventoryHostsStrings.get('deletegroup.HOST', $scope.toDelete.total_hosts);
-
-                if($scope.toDelete.total_groups === 0 || $scope.toDelete.total_hosts === 0) {
-                    if($scope.toDelete.total_groups === 0) {
-                        $scope.strings.deleteModal.deleteGroupsHosts = InventoryHostsStrings.get('deletegroup.DELETE_HOST', $scope.toDelete.total_hosts);
-                        $scope.strings.deleteModal.promoteGroupsHosts = InventoryHostsStrings.get('deletegroup.PROMOTE_HOST', $scope.toDelete.total_hosts);
+            const promises = [];
+
+            Rest.setUrl(group.related.hosts);
+            promises.push(Rest.get());
+
+            Rest.setUrl(group.related.children);
+            promises.push(Rest.get());
+
+            Promise.all(promises)
+                .then(([hostResponse, groupResponse]) => {
```

Comment:
Removing the `total_hosts` and `total_groups` counts means we're no longer showing the counts on the delete warnings. Instead, a more generic message is used.

Example 1 (one group, multiple hosts):
![Screenshot from 2019-12-10 11-59-28](https://user-images.githubusercontent.com/9753817/70551525-aa2d2480-1b45-11ea-9e18-5da5a63bd99e.png)

Example 2 (no hosts or groups):
![Screenshot from 2019-12-10 12-00-23](https://user-images.githubusercontent.com/9753817/70551544-b44f2300-1b45-11ea-8975-f07b21cdcf88.png)